packages/dashmate/README.md

```diff
@@ -77,33 +77,34 @@ The package contains a CLI, Docker Compose and configuration files.
 
 The CLI can be used to perform routine tasks. Invoke the CLI with `dashmate` if linked during installation, or with `node bin/dashmate` if not linked. To list available commands, either run `dashmate` with no parameters or execute `dashmate help`. To list the help on any command just execute the command, followed by the `--help` option.
 
-### Setup node
+### Set up node
 
 The `setup` command is used to quickly configure common node configurations. Arguments may be provided as options, otherwise they will be queried interactively with sensible values suggested.
 
 ```
+Set up a new Dash node
+
 USAGE
-  $ dashmate setup [PRESET] [NODE-TYPE]
+  $ dashmate setup [PRESET] [-v] [-d] [-c <value>] [-m <value>]
 
 ARGUMENTS
-  PRESET     (testnet|local) Node configuration preset
-  NODE-TYPE  (masternode|fullnode) Node type
+  PRESET  (mainnet|testnet|local) Node configuration preset
 
-OPTIONS
-  -d, --[no-]debug-logs                                    enable debug logs
-  -i, --external-ip=external-ip                            external ip
-  -k, --operator-bls-private-key=operator-bls-private-key  operator bls private key
-  -m, --miner-interval=miner-interval                      interval between blocks
-  -p, --funding-private-key=funding-private-key            private key with more than 1000 dash for funding collateral
-  -v, --verbose                                            use verbose mode for output
-  --node-count=node-count                                  number of nodes to setup
+FLAGS
+  -c, --node-count=<value>      number of nodes to set up
+  -d, --[no-]debug-logs         enable debug logs
+  -m, --miner-interval=<value>  interval between blocks
+  -v, --verbose                 use verbose mode for output
+
+DESCRIPTION
+  Set up a new Dash node
 ```
 
 Supported presets:
  * `testnet` - a masternode or full node for testnet
```

Comment:
Should be any entry here for `mainnet` now too?

Reply:
Yeah, please update readme )))